dovecot-pigeonhole: fails to compile when full language support is enabled

Maintainer: @MikePetullo
Environment: latest version in the trunk, OpenWrt r7530-7c306ae

Description:

dovecot-pigeonhole fails to compile if the full language support is enabled (CONFIG_BUILD_NLS is set to 'y'). The error message is (paths truncated):
```
<snip>arm-openwrt-linux-muslgnueabi/bin/ld: warning: libiconv.so.2, needed by <snip>staging_dir/target-arm_cortex-a9_musl_eabi/usr/lib/dovecot//libdovecot-storage.so, not found (try using -rpath or -rpath-link)
<snip>staging_dir/target-arm_cortex-a9_musl_eabi/usr/lib/dovecot//libdovecot.so: undefined reference to `libiconv'
<snip>staging_dir/target-arm_cortex-a9_musl_eabi/usr/lib/dovecot//libdovecot.so: undefined reference to `libiconv_open'
<snip>staging_dir/target-arm_cortex-a9_musl_eabi/usr/lib/dovecot//libdovecot.so: undefined reference to `libiconv_close'
collect2: error: ld returned 1 exit status
```

It appears that including nls.mk in Makefile fixes the problem.

--- a/mail/pigeonhole/Makefile
+++ b/mail/pigeonhole/Makefile
@@ -11,7 +11,7 @@ PKG_NAME:=dovecot-pigeonhole
 PKG_VERSION_PLUGIN:=0.5.2
 PKG_VERSION_DOVECOT:=$(shell make --no-print-directory -C ../dovecot/ val.PKG_VERSION V=s)
 PKG_VERSION:=$(PKG_VERSION_DOVECOT)-$(PKG_VERSION_PLUGIN)
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 DOVECOT_VERSION:=2.3
 
@@ -25,6 +25,7 @@ PKG_BUILD_DIR:=$(BUILD_DIR)/dovecot-$(DOVECOT_VERSION)-pigeonhole-$(PKG_VERSION_
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/dovecot-pigeonhole
   SECTION:=mail

